### ISSUE TITLE
KS0 upgrading from 140 to 150 has no effect on hashrate.

### ISSUE THREAD

**Issue by defactoambassador** [incidrthreat/iceriver_oc#7]
Hello. One of my KS0 machines was using 140 OC. I decided to upgrade to 150 a few hours ago, and left it running. 

It's been about 3 hours now but the hashrate is unchanged from before (a few days average of 133-134GH). It should have at least increased to 140~, but it doesn't reach more than 134GH. The temperature increased from 61 to 64 however, so more power is definitely being drawn. 

I'm using 120W power and herominer pool. What could be the cause?

**Comment by incidrthreat** [comment]
This is unfortunate to hear, sometimes the silicon lotto doesn't bless us and the core is being locked bc of it.  What is your efficiency when it comes to the OC change?

**Comment by Topbcy** [comment]
change your psu, casue 120w issue, 
120w psu only work with 140g firmware.
if you want able to using high hashrate firmware,you need change to 150w at least.

**Comment by defactoambassador** [comment]
> This is unfortunate to hear, sometimes the silicon lotto doesn't bless us and the core is being locked bc of it. What is your efficiency when it comes to the OC change?

I don't have equipment that can measure wall power draw to calculate the efficiency. The heat efficiency though is somewhat limiting (3~4 degrees increase for 10W theoretical difference).

Were we supposed to reset to factory setting first before upgrading to another firmware? I rolled back to the latest iceriver firmware and then tried applying the 150GH patch. It didn't work, still showing <140 hashrate.

**Comment by defactoambassador** [comment]
> change your psu, casue 120w issue, 120w psu only work with 140g firmware. if you want able to using high hashrate firmware,you need change to 150w at least.

I have two KS0 machines with 120W PSU, the other one running on stock firmware. I will upgrade the stock one to 150GH and then compare the results with the 140 machine.

**Comment by Topbcy** [comment]
Because I same with you about your issue. I have 120w psu 19V,I using 160g fw, I am only get 125G hashrate. and my watt show 110-115, I have same with your thinking.So I flash to my ks0 using 140G fw and psu show 95-100 watts ,well I can get 140G hashrate. After all I buy a new one pus 230w, that fix all my issue.

and flash mod firmware no need flash offical fw before. if you want change fw, just flash what you want,but fisrt flash you need notice you already is offical newest fw.

**Comment by Topbcy** [comment]
I think 120w psu not really can full using 120, maybe only 110w
and all chip oc about 160g, maybe you ks0 need 125w, some people just need 110w, sorry my english, I just want to say not all chip(ks0) have same OC ability, like CPU.

At all if you change you psu to higher watt,but still cant get higher hashrate, unfortunately .. you only can use lower hashrate firmware.

**Comment by defactoambassador** [comment]
> Because I same with you,about your issue. I have 120w psu 19V,I using 160g fw, I am only get 125G hashrate. and my watt show 110-115, I have same with your thinking.So I flash to my ks0 using 140G fw and psu show 95-100 watts ,well I can get 140G hashrate. After all I buy a new one pus 230w, that fix all my issue.
> 
> and flash mod firmware no need flash offical fw before. if you want change fw, just flash what you want,but fisrt flash you need notice you already is offical newest fw.

Okay, thanks for checking. 140GH is okay since it has less power draw and I don't need another PSU or cooling fan. I will get a better PSU and cooling solution for 160GH.

**Comment by JustLook12** [comment]
> change your psu, casue 120w issue, 120w psu only work with 140g firmware. if you want able to using high hashrate firmware,you need change to 150w at least.

that is not correct. I have runned three KS0 machines succesfully on 160 Gh/s with 120W PSU. Note i did update to larger PSU

**Comment by Topbcy** [comment]
> > change your psu, casue 120w issue, 120w psu only work with 140g firmware. if you want able to using high hashrate firmware,you need change to 150w at least.
> 
> 
> 
> that is not correct. I have runned three KS0 machines succesfully on 160 Gh/s with 120W 

I mean not all people's ks0 have same OC ability, just like desktop CPU oc.

Some peopel need oc 160g.need more watt, and you using 120w psu, that is not enough.
And in my experience, change psu fix my issue about using 160G fw cant get 160G hashrate

**Comment by Topbcy** [comment]
> > Because I same with you,about your issue. I have 120w psu 19V,I using 160g fw, I am only get 125G hashrate. and my watt show 110-115, I have same with your thinking.So I flash to my ks0 using 140G fw and psu show 95-100 watts ,well I can get 140G hashrate. After all I buy a new one pus 230w, that fix all my issue.
> 
> > 
> 
> > and flash mod firmware no need flash offical fw before. if you want change fw, just flash what you want,but fisrt flash you need notice you already is offical newest fw.
> 
> 
> 
> Okay, thanks for checking. 140GH is okay since it has less power draw and I don't need another PSU or cooling fan. I will get a better PSU and cooling solution for 160GH.

yes higher hashrate need more cooling

**Comment by Topbcy** [comment]
At all if you change you psu to higher watt,but still cant get higher hashrate, unfortunately .. you only can use lower hashrate firmware.

**Comment by JustLook12** [comment]
> > > change your psu, casue 120w issue, 120w psu only work with 140g firmware. if you want able to using high hashrate firmware,you need change to 150w at least.
> > 
> > 
> > that is not correct. I have runned three KS0 machines succesfully on 160 Gh/s with 120W
> 
> I mean not all people's ks0 have same OC ability, just like desktop CPU oc.
> 
> Some peopel need oc 160g.need more watt, and you using 120w psu, that is not enough. And in my experience, change psu fix my issue about using 160G cant get 160G hashrate

fair point!

**Comment by Topbcy** [comment]
> > > > change your psu, casue 120w issue, 120w psu only work with 140g firmware. if you want able to using high hashrate firmware,you need change to 150w at least.
> 
> > > 
> 
> > > 
> 
> > > that is not correct. I have runned three KS0 machines succesfully on 160 Gh/s with 120W
> 
> > 
> 
> > I mean not all people's ks0 have same OC ability, just like desktop CPU oc.
> 
> > 
> 
> > Some peopel need oc 160g.need more watt, and you using 120w psu, that is not enough. And in my experience, change psu fix my issue about using 160G cant get 160G hashrate
> 
> 
> 
> fair point!

wish all people like you lucky.
160G easy lower watts~~

**Comment by NeroNuke** [comment]
I noticed that the update only works if you reset the miner to factory settings and then update with the OC file. BUT, 150 and 160 files both do 160GH/s for me, and I noticed that the filesize for these two is identical?

**Comment by defactoambassador** [comment]
> I noticed that the update only works if you reset the miner to factory settings and then update with the OC file. BUT, 150 and 160 files both do 160GH/s for me, and I noticed that the filesize for these two is identical?

I tried factory resetting and then reupdating the OC files. Same result as before.

I'll replace the power supply now see if it changes anything. Btw, the other KS0 machine running on stock 120W upgraded to 160 GH no problem. Very good temps too on 100% fan (56 degrees) in autumn weather.

**Comment by NeroNuke** [comment]
Hi,I hold down the button for around 20 sec until the led starts blinking…Sent from my iPhoneOn Oct 4, 2023, at 12:43, defactoambassador ***@***.***> wrote:﻿

I noticed that the update only works if you reset the miner to factory settings and then update with the OC file. BUT, 150 and 160 files both do 160GH/s for me, and I noticed that the filesize for these two is identical?

I tried factory resetting and then reupdating the OC files. Same result as before.
I'll replace the power supply now see if it changes anything. Btw, the other KS0 machine running on stock 120W upgraded to 160 GH no problem. Very good temps too on 100% fan (56 degrees) in autumn weather.
